gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,419 @@
+            "id": 3,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Health",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 20
+        },
+        {
+            "check": "gitlab.prometheus_endpoint_up",
+            "group_by": [],
+            "grouping": "cluster",
+            "height": 8,
+            "id": 4,
+            "tags": [
+                "*"
+            ],
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Endpoint Up",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 29
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "18",
+            "height": 25,
+            "html": "Service Checks",
+            "id": 5,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "right",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 15,
+            "x": 1,
+            "y": 12
+        },
+        {
+            "height": 13,
+            "id": 6,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.rack.http_requests_total{*} by {method}.as_count()",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "bars"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "Http requests",
+            "type": "timeseries",
+            "width": 43,
+            "x": 34,
+            "y": 13
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "18",
+            "height": 10,
+            "html": "Load Information",
+            "id": 7,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 43,
+            "x": 34,
+            "y": 1
+        },
+        {
+            "height": 13,
+            "id": 8,
+            "legend": false,
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:gitlab.pipelines_created_total{*}",
```

Comment:
- ditto template variables
- Let's switch `avg` to `sum`
- Shouldn't that be a count? Can we add `as_count()` to the end of that?